lib/keycloak.rb

```diff
@@ -210,14 +210,16 @@ def self.get_token_introspection(token = '', client_id = '', secret = '', intros
       exec_request _request
     end
 
-    def self.url_login_redirect(redirect_uri, response_type = 'code', client_id = '', authorization_endpoint = '')
+    def self.url_login_redirect(redirect_uri, response_type = 'code', client_id = '', authorization_endpoint = '', scope: nil)
```

Comment:
scope should be an array because we can request for multiple scopes ... for example scope = [email,openid]